Makefile

```diff
@@ -10,6 +10,7 @@ HELPERS := assert_device_present \
 	   assert_partition_found \
 	   assert_sysfs_attr_present \
 	   bootrr \
+	   bootrr-auto \
```

Comment:
something similar is available in https://github.com/andersson/bootrr/pull/19/. we need one or the other..
@alimon  can you please review this one?

Reply:
If #19 is merged I can rebase my patches above.